tests/Aspire.Dashboard.Tests/FormatHelpersTests.cs

```diff
@@ -30,4 +30,45 @@ public void FormatNumberWithOptionalDecimalPlaces_GermanCulture(string expected,
     {
         Assert.Equal(expected, FormatHelpers.FormatNumberWithOptionalDecimalPlaces(value, CultureInfo.GetCultureInfo("de-DE")));
     }
+
+    [Theory]
+    [InlineData("06/15/2009 13:45:30.000", true, "2009-06-15T13:45:30.0000000Z")]
+    [InlineData("06/15/2009 13:45:30.123", true, "2009-06-15T13:45:30.1234567Z")]
+    [InlineData("06/15/2009 13:45:30", false, "2009-06-15T13:45:30.0000000Z")]
+    [InlineData("06/15/2009 13:45:30", false, "2009-06-15T13:45:30.1234567Z")]
```

Comment:
You could include the culture name in the data here, and collapse some tests into one. Invariant culture is just `en-US`.

Reply:
I didn't want the test to be too big. This is a balance between dozens and one.